app/assets/javascripts/katello/hosts/host_and_hostgroup_edit.js

```diff
@@ -267,11 +267,10 @@ KT.hosts.on_synced_content_dropdown_change = function() {
 KT.hosts.set_install_media_bindings = function() {
     // reset the host medium id
     $("#host_medium_id").on("change", KT.hosts.on_install_media_dropdown_change);
```

Comment:
I think this should be changed as well, without this it's still broken :/
```suggestion
    $("#host_medium_id").on("select2:select", KT.hosts.on_install_media_dropdown_change);
```